meetup/migrations/0002_auto_20160307_1925.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.2 on 2016-03-07 19:25
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import meetup.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('meetup', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='meetup',
+            name='meetup_link',
+            field=models.URLField(default=None, max_length=500, unique=True, validators=[meetup.models.validate_urlname, meetup.models.validate_meetup_exists]),
```

Comment:
Try squashing this migration into the previous one. Sort of like squashing git commits, Django provides a migration squashing tool: https://docs.djangoproject.com/en/1.9/topics/migrations/#squashing-migrations